.github/workflows/vuln.yml

```diff
@@ -0,0 +1,28 @@
+name: Vulnerability Check
```

Comment:
Since this has exactly the same triggers as the `Code Scanning` workflow, I would say that this could be folded in as a separate job under that workflow, rather than as a workflow of its own. At the end of the day, they both perform static analysis to find vulnerabilities